src/components/frame/Extensions/layouts/utils.ts

```diff
@@ -1,7 +1,22 @@
 import { NETWORK_DATA } from '~constants';
+import { NavigationSidebarLinksListItem } from '~v5/frame/NavigationSidebar/partials/NavigationSidebarLinksList/types';
 
 export const getChainIconName = (
   chainId: number | undefined,
 ): string | undefined =>
   Object.values(NETWORK_DATA).find(({ chainId: id }) => id === chainId)
     ?.iconName;
+
+export const checkIfIsActive = (
+  pathname: string | undefined,
+  items: NavigationSidebarLinksListItem[],
+): boolean => {
+  if (!pathname) {
+    return false;
+  }
+
+  return items.some(
+    ({ to }) =>
+      (typeof to === 'string' ? to.replace(/^\/|/g, '') : to) === pathname,
```

Comment:
Are you sure you want to remove all slashes?

Reply:
This regex is to remove slash from the start of the string